GA-worksheet.md

```diff
@@ -75,21 +75,21 @@
 
 6. What does each digit of this Hexidecimal Color Code represent? #E78
 
- E represents ...
+ E represents ... the amount of redness
 
- 7 represents ...
+ 7 represents ... the amount of greeness
 
- 8 represents ...
+ 8 represents ... the amount of blueness 
 
-7. What is the Hex Color Code for true purple?  #
+7. What is the Hex Color Code for true purple?  #990099
 
-8. _______________ is the programming language that allows websites to be interactive.
+8. JAVASCRIPT is the programming language that allows websites to be interactive.
 
-9. This tag allows us to add JavaScript into any webpage? <          >
+9. This tag allows us to add JavaScript into any webpage? <script>
 
-10. What is an event?
+10. What is an event? something that happens everytime you move your mouse, click something, mouse over things, press a key or scroll. by default they go unnotticed.
 
-11. How does JavaScript use events?
+11. How does JavaScript use events?java can listen for specific events and sort of create reactions to them.
```

Comment:
Just wanted to point out that Java and JavaScript are similarly named, but completely different languages.